management/src/main/java/org/kaazing/gateway/management/context/DefaultManagementContext.java

```diff
@@ -620,11 +638,28 @@ private ServiceDefaultsConfigurationBean addServiceDefaultsConfigurationBean(Ser
     }
 
     private ManagementFilter addManagementFilter(ServiceContext serviceContext, ServiceManagementBean serviceBean) {
-        ManagementFilter managementFilter = new ManagementFilter(serviceBean);
+        ManagementFilter managementFilter = new ManagementFilter(serviceBean,
+                                                                 monitoringEntityFactory,
```

Comment:
What happens if monitoringEntityFactory is null here?  Maybe call a different constructor?